packages/core/src/civil.ts

```diff
@@ -123,12 +123,12 @@ export class Civil {
    * @returns A singleton instance of TCR living on the current network
    * @throws {CivilErrors.UnsupportedNetwork} In case we're trying to get a non-deployed singleton
    */
-  public tcrSingletonTrusted(): OwnedAddressTCRWithAppeals {
-    return OwnedAddressTCRWithAppeals.singleton(this.web3Wrapper, this.contentProvider);
+  public tcrSingletonTrusted(): CivilTCR {
+    return CivilTCR.singleton(this.web3Wrapper, this.contentProvider);
   }
 
   /**
-   * Returns the EIP20 instance associated with the deployed OwnedAddressTCRWithAppeals
+   * Returns the EIP20 instance associated with the deployed CivilTCR
    */
   public async getEIP20ForDeployedTCR(): Promise<EIP20> {
```

Comment:
by the way, why the helper functions. Might as well get this stuff from the tcr?